core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -269,35 +266,36 @@ object AclCommand {
 
   class AclCommandOptions(args: Array[String]) {
     val parser = new OptionParser(false)
-    val authorizerOpt = parser.accepts("authorizer", "Fully qualified class name of the authorizer, defaults to kafka.security.auth.SimpleAclAuthorizer.")
+    val authorizerOpt = parser.accepts("authorizer", "Fully qualified class name of the Authorizer, defaults to kafka.security.auth.SimpleAclAuthorizer.")
       .withRequiredArg
-      .describedAs("authorizer")
+      .describedAs("class name of authorizer")
       .ofType(classOf[String])
       .defaultsTo(classOf[SimpleAclAuthorizer].getName)
 
-    val authorizerPropertiesOpt = parser.accepts("authorizer-properties", "REQUIRED: properties required to configure an instance of Authorizer. " +
-      "These are key=val pairs. For the default authorizer the example values are: zookeeper.connect=localhost:2181")
+    val authorizerPropertiesOpt = parser.accepts("authorizer-properties", "Properties required to configure an instance of Authorizer. " +
+      "These are key=val pairs. For the default authorizer, the example values are: zookeeper.connect=localhost:2181.")
       .withRequiredArg
-      .describedAs("authorizer-properties")
+      .describedAs("properties(key=value pairs) for configuring instance of Authorizer")
```

Comment:
I think we should be careful with the length of the these descriptions because they appear in the help doc of the command. A long description could make the command help difficult to read. It would be great if we can make them more descriptive by using just a few words. If not, we can keep them as is.